### PR TITLE
Update guide.md

### PR DESCRIPTION
**Pull request by PanosPeriorellis**
## Purpose

* ...

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[* ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: samples/batch/batch-ingestion-client/PowerBI/guide.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 
-# Visualizing Outputs of the Accelerator in Power BI
+# Visualizing the transcription output in Power BI
 
 ## Setup Guide
 
-The following guide will help you create Power BI reports from templates that will help to realize value from the speech transcription accelerator you just deployed.
+The following guide will help you create Power BI reports from templates that will help to realize value from the speech transcription out.
```
Comment (chlandsi) [inline samples/batch/batch-ingestion-client/PowerBI/guide.md:6]: output?

File: samples/batch/batch-ingestion-client/PowerBI/README.md [hunk 2]
```diff
@@ -8,30 +8,30 @@ Guidance on onboarding samples to docs.microsoft.com/samples: https://review.doc
...
 ## Contents
 
-There are template reports provided for two use cases that help to realize value of the speech solution accelerator. Below are the file names and a brief description of each template.
+These are template reports provided for two use cases that help realize value of the transcription output. Below are the file names and a brief description of each template.
 
 | File       | Description                                |
 |-------------------|--------------------------------------------|
-| `SpeechInsights.pbit`       | A collection of reports that use data from Transcription Enabled Storage to quantify your customer care agents' interactions and improve empathetic conversations.                       |
-| `SentimentInsights.pbit`      | A collection of reports that use data from Transcription Enabled Storage to understand the sentiment of your customers during an interaction and improve satisfaction.      |
+| `SpeechInsights.pbit`       | A collection of reports that use transcrpition data to quantify your customer care agents' interactions and improve empathetic conversations.                       |
```
Comment (chlandsi) [inline samples/batch/batch-ingestion-client/PowerBI/README.md:21]: typo

Comment (chlandsi) [reply samples/batch/batch-ingestion-client/PowerBI/README.md:21]: also below

File: samples/batch/batch-ingestion-client/PowerBI/guide.md (hunk 1 above)
Comment (zhouwangzw) [inline samples/batch/batch-ingestion-client/PowerBI/guide.md:6]: 2 "Will help" in the same sentences make it difficult to read. :-)

**Review by dargilco — DISMISSED** [review]

File: samples/batch/batch-ingestion-client/Setup/guide.md [hunk 3]
```diff
@@ -2,11 +2,11 @@
 
 Get your audio files automatically transcribed without writing any code what-so-ever! The Batch Ingestion Client will monitor your dedicated Azure Storage container so that new audio files are transcribed automatically as soon as they land.
 
-The simplest way to explain this tech is as a zero-touch transcription solution for all your audio files in your Azure Storage. If you are looking for a quick and effortless way to transcribe your audio files or even explore transcription, without writing any code, then this solution is for you. Through an ARM template deployment, all the resources necessary to seamlessly process your audio files are set-up and set in motion.
+The simplest way to explain this tech is as a zero-touch transcription solution for all your audio files in your Azure Storage. If you are looking for a quick and effortless way to transcribe your audio files or even explore transcription, then this solution is for you.
```
Comment (v-demjoh) [inline samples/batch/batch-ingestion-client/Setup/guide.md:6]: I've written a new introduction to replace this entire first section. Is this correct and complete?:

This Batch Ingestion Client example transcribes all audio files found in your Azure Storage. By monitoring your Azure Storage container, this example can transcribe each new audio file automatically immediately after its upload to completes.

File: samples/batch/batch-ingestion-client/PowerBI/guide.md (hunk 1 above)
Comment (PanosPeriorellis) [reply samples/batch/batch-ingestion-client/PowerBI/guide.md:6]: good point. address

Comment (PanosPeriorellis) [reply samples/batch/batch-ingestion-client/PowerBI/guide.md:6]: address

File: samples/batch/batch-ingestion-client/PowerBI/README.md (hunk 2 above)
Comment (PanosPeriorellis) [reply samples/batch/batch-ingestion-client/PowerBI/README.md:21]: noted, and addressed

File: samples/batch/batch-ingestion-client/Setup/guide.md (hunk 3 above)
Comment (PanosPeriorellis) [reply samples/batch/batch-ingestion-client/Setup/guide.md:6]: yes, i will use it differently

File: samples/batch/batch-ingestion-client/Setup/guide.md [hunk 4]
```diff
@@ -97,19 +97,15 @@ resources within the same resource group so we suggest you create a new resource
...
 The following settings all relate to the resources and their attributes
 
-
 * Give your storage account a name [you will be using a new storage
 account rather than an existing one].
 
 The following 2 steps are optional. Omitting them will result in using the base model to obtain
 transcripts. If you have created a Speech model, then
 
+* Enter optionally an Acoustic model
```
Comment (HenryvanderVegte) [inline samples/batch/batch-ingestion-client/Setup/guide.md:106]: Rather say custom model or speech recognition model

**Review by HenryvanderVegte — DISMISSED** [review]

**Review by HenryvanderVegte — APPROVED** [review]

**Pull request merged**